changelog.md

```diff
@@ -27,6 +27,14 @@ Event-proxying has the following deprecation schedule:
 - 1.7.x - Standby Marathon instances return redirect responses. The old behavior of proxying event streams can be brought back with the command-line argument `--deprecated_features=proxy_events`.
 - 1.8.x - Event stream proxying logic will be completely removed. If `--deprecated_features=proxy_events` is still specified, Marathon will refuse to launch, with an error.
 
+### Maintenance Mode Support Production Ready, Now Default
+
+Support for declining offers for agents undergoing a maintenance window is now enabled by default, and the feature is now recommended for production use.
```

Comment:
would it make sense to define what we mean specifically for "maintenance window".    Specific to Mesos Maintenance Window http://mesos.apache.org/documentation/latest/maintenance/#how-does-it-work

We decline offers for agents in DRAIN mode.    Offers for agents that are in DOWN mode do not send offers.

Reply:
Thanks for this, this is a great clarification to make